AllReadyApp/Web-App/AllReady/Areas/Admin/Models/TaskSummaryModel.cs

```diff
@@ -36,5 +37,6 @@ public class TaskSummaryModel
         public List<VolunteerModel> AllVolunteers { get; set; } = new List<VolunteerModel>();
 
         public int AcceptedVolunteerCount => AssignedVolunteers?.Where(v => v.HasVolunteered).Count() ?? 0;
+        public int NumberOfVolunteersSignedUp { get; set; }

```

Comment:
Is this different than `AllVolunteers.Count()`? How is this set/updated differently?